pkgs/retry/retry

```diff
@@ -4,6 +4,8 @@
 #
 set -euf
 
+export PATH=@coreutils@/bin"${PATH+:$PATH}"
```

Comment:
Beside the syntax, it has nothing to do with Makefile. Have a look at [the documentation of substituteAll](https://nixos.org/nixpkgs/manual/#fun-substituteAll) and, to see how every attribute (e.g. [coreutils](https://github.com/zalora/microgram/pull/16/files#diff-9d168b32227c60d3951d0fd8cf125d22R7)) is passed as an environment variable to the builder, see [NixOS's derivation section](http://nixos.org/nix/manual/#ssec-derivation).